<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TranslationTextFileDialog</class>
 <widget class="QDialog" name="TranslationTextFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>265</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSelectFile">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>176</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Output Directory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOpenAndProcess">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>235</y>
     <width>266</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Directory And Process</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leDirName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>195</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>325</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_cWEBDB">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>90</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>/drop-folder/Dropbox/NewBible/cWeb.bbl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWebDB">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>45</y>
     <width>176</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select cWeb</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>125</y>
     <width>366</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="leBookName">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>290</y>
     <width>206</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>50</y>
     <width>191</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbTagalogIlongo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>285</y>
     <width>126</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tagalog/Ilonggo</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbTaglish">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>285</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Taglish</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
